src/base/bittorrent/session.cpp

```diff
@@ -2048,8 +2053,8 @@ void Session::bottomTorrentsPriority(const QStringList &hashes)
         torrentQueue.pop();
     }
 
-    foreach (const InfoHash &hash, m_loadedMetadata.keys())
-        torrentQueuePositionBottom(m_nativeSession->find_torrent(hash));
+    for (auto i = m_loadedMetadata.cbegin(); i != m_loadedMetadata.cend(); ++i)
+        torrentQueuePositionBottom(m_nativeSession->find_torrent(i.key())); // i.key() <=> InfoHash
```

Comment:
maybe `// decltype(i.key()) = InfoHash`, or just remove the comment.

Reply:
just remove the comment